integration_test/src/test_runner/mod.rs

```diff
@@ -743,7 +743,7 @@ fn get_column_count(stmt: &Statement<Allocated, NoResult, AutocommitOn>) -> Resu
 fn fetch_row(stmt: &Statement<Allocated, NoResult, AutocommitOn>) -> Result<bool> {
     unsafe {
         match odbc_sys::SQLFetch(stmt.handle() as HStmt) {
-            SqlReturn::SUCCESS => Ok(true),
+            SqlReturn::SUCCESS | SqlReturn::SUCCESS_WITH_INFO => Ok(true),
```

Comment:
Note: added this to account for updates to `SQLFetch`. the `baz` view in the `integration_test` database triggers the newly added `SUCCESS_WITH_INFO` where applicable.

Reply:
umh, why is baz generating a SUCCESS_WITH_INFO right now? Is there a deserialization error or is it something else?

Reply:
I think it is a deserialization error that was just getting skipped over before, but I see ```Err(OdbcFunctionFailed("SQLFetch", "SUCCESS_WITH_INFO", "[MongoDB][Core] missing field `version`"))``` when I run this on my windows host with these changes.